wolfcrypt/src/port/nxp/dcp_port.c

```diff
@@ -35,6 +35,8 @@
 #include <wolfssl/wolfcrypt/sha.h>
 #include <wolfssl/wolfcrypt/sha256.h>
 #include <wolfssl/wolfcrypt/error-crypt.h>
+#include "fsl_device_registers.h"
```

Comment:
Please move these into the `#ifdef WOLFSSL_IMXRT_DCP` section, otherwise will produce build errors on systems including all .c files.

Reply:
fixed, thanks